src/utils/http.ts

```diff
@@ -10,8 +18,16 @@ export async function fetchProduct(id: string): Promise<Product> {
     return await response.json();
 }
 
-export async function fetchProducts() {
-    const response = await fetch('https://ma-backend-api.mocintra.com/api/v1/products');
+export async function fetchProducts(
+    limit: number = 0,
+    offset: number = 0,
+    title: string = '',
+    sortOrder: SortByEnum = SortByEnum.HIGH_TO_LOW,
+    categoryId: string = '',
+): Promise<FetchProductsInterface> {
```

Comment:
if u have more than 2 params in a function, it is a good practise to replace parameters with an object